README.md

```diff
@@ -42,6 +42,17 @@ github "GetStream/stream-swift"
 ```
 Then run `carthage update`.
 
+Then drag the built `.framework` binaries from `Carthage/Build/<platform>` into your application’s Xcode project.
+```
+GetStream.framework
+Faye.framework
+Alamofire.framework
+Moya.framework or RxMoya.framework or ReactiveMoya.framework
```

Comment:
Hi @nimajalali !
Thanks for your PR. Could you please fix README file, Stream SDK doesn't use Rx/Reactive Moya components, only Moya framework itself. And, please, add bullets to the list.

```
Then drag the built `.framework` binaries from `Carthage/Build/<platform>` into your application’s Xcode project:
- GetStream.framework
- Faye.framework
- Alamofire.framework
- Moya.framework
- Starscream.framework
- Swime.framework
- Result.framework
```